nautobot/core/graphql/schema.py

```diff
@@ -381,30 +362,27 @@ def extend_schema_type_relationships(schema_type, model):
     return schema_type
 
 
-def generate_query_mixin():
-    """Generates and returns a class definition representing a GraphQL schema."""
-
-    class_attrs = {}
-
-    def already_present(model):
-        """Check if a model and its resolvers are staged to added to the Mixin."""
-
-        single_item_name = str_to_var_name(model._meta.verbose_name)
-        list_name = str_to_var_name(model._meta.verbose_name_plural)
-
-        if single_item_name in class_attrs:
-            logger.warning(
-                f"Unable to register the schema single type '{single_item_name}' in GraphQL, "
-                f"there is already another type {class_attrs[single_item_name]._type} registered under this name"
-            )
-            return True
-
-        if list_name in class_attrs:
-            logger.warning(
-                f"Unable to register the schema list type '{list_name}' in GraphQL, "
-                f"there is already another type {class_attrs[list_name]._type} registered under this name"
-            )
-            return True
+def make_class(cls):
```

Comment:
Can you add a docstring explaining why this function is needed and what it does?

Reply:
I have been digging into this more deeply and I think we need to rethink this. This results in the formal type names being `WhateverCopy` which is not great...